pinot-broker/src/main/java/org/apache/pinot/broker/routing/builder/PartitionAwareOfflineRoutingTableBuilder.java

```diff
@@ -179,7 +180,8 @@ public synchronized void computeOnExternalViewChange(String tableName, ExternalV
   }
 
   /**
-   * Get partition id from segment Zk metadata. This helper function assumes there is only one column partition metadata.
+   * Get partition id from segment Zk metadata.
+   * TODO: This helper function assumes the segment is partitioned on at most one column, and contains only 1 partition.
```

Comment:
I guess that removing the above assumption is `TODO` task? Can you update the comment what needs to be resolved?

Reply:
Done.